libs/labelbox/src/labelbox/client.py

```diff
@@ -938,10 +939,12 @@ def rootSchemaPayloadToFeatureSchema(client, payload):
                                    rootSchemaPayloadToFeatureSchema,
                                    ['rootSchemaNodes', 'nextCursor'])
 
-    def create_ontology_from_feature_schemas(self,
-                                             name,
-                                             feature_schema_ids,
-                                             media_type=None) -> Ontology:
+    def create_ontology_from_feature_schemas(
+            self,
+            name,
+            feature_schema_ids,
+            media_type: MediaType = None,
+            editor_task_type: EditorTaskType = None) -> Ontology:
```

Comment:
Docstring here too.

Reply:
@vbrodsky We'd really like to hide this `editor_task_type` from the customer. One idea we had was to add a new `media_type` called `"model-chat-evaluation"` which is a special media type that maps to `MediaType.Conversational` and an editor task type with value `model-chat-evaluation`? What does you think?

cc @lgluszek